### PR TITLE
Warning: 3 references not resolved

### PR DESCRIPTION
**Pull request by sidkcr**
# Fixes # 
#69 - Warning: 3 References not resolved

### Checklist
- [x] I have read the [Contribution Guide] and my PR follows them.
- [x] I updated my branch with the master branch.

### Short description of what this PR does:
- This resolves build warnings of the solution
- Also, I've changed year in LICENSE.txt to 2018 to make tests successful. (I'll revert this change if it's not needed)

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/smtpapi-csharp?pullRequest=73) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/smtpapi-csharp?pullRequest=73) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/smtpapi-csharp?pullRequest=73) it.</sub>

**Comment by thinkingserious** [comment]
Thanks @sidkcr! I've added this to our backlog for a code review.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 # Change Log
 All notable changes to this project will be documented in this file.
 
+[2020-08-19] Version 1.3.9
```
Comment (thinkingserious) [inline CHANGELOG.md:4]: Please remove entries here, these are now auto-generated.

**Comment by thinkingserious** [comment]
Hello @sidkcr,

I would like to get this one updated to merge. Are you still interested in working on this PR? If not, we can close it out and add to our backlog. Thank you!

With best regards,

Elmer

File: CHANGELOG.md (hunk 1 above)
Comment (sidkcr) [reply CHANGELOG.md]: @thinkingserious done

**Comment by sidkcr** [comment]
@thinkingserious updated branch and resolved review comments

**Review by JenniferMah — COMMENTED** [review]
Hi @sidkcr could you please resolve the merge conflicts in the following files?
`Smtpapi/HeaderTests/SendGrid.SmtpApi.HeaderTests.csproj`
`Smtpapi/HeaderTests/packages.config`
`Smtpapi/Smtpapi/SendGrid.SmtpApi.csproj`

**Comment by thinkingserious** [comment]
Closing until PR feedback is addressed.

**Pull request closed without merging**